packages/lesswrong/server/scripts/convertImagesToCloudinary.ts

```diff
@@ -195,8 +196,10 @@ export async function convertImagesInObject(
     
     const latestRev = await getLatestRev(_id, fieldName);
     if (!latestRev) {
-      // eslint-disable-next-line no-console
-      console.error(`Could not find a latest-revision for ${collectionName} ID: ${_id}`);
+      if (!isAnyTest) {
+        // eslint-disable-next-line no-console
+        console.error(`Could not find a latest-revision for ${collectionName} ID: ${_id}`);
```

Comment:
I think this might actually be a bug FWIW. I would leave this, unless you know why it's happening.

Reply:
It happens because we have mock data in the database for Cypress that has `contents` fields which aren't backed by any actual data in the `Revisions` table. It's been happening for a long time but I only just got annoyed by it enough to actually do something about it.

Reply:
Oh, right.